controllers/argocd/deployment.go

```diff
@@ -1293,6 +1293,13 @@ func isDexDisabled() bool {
 	return false
 }
 
+func removeManagedByLabelOnArgoCDDeletion() bool {
```

Comment:
can we call this `isRemoveManagedByLabelOnArgoCDDeletion()` or something similar instead?
May be better if the purpose of the function is more clear from just reading the name IMO